packages/python/agentmemory/__init__.py

```
@@ -0,0 +1,469 @@
+
+        _, body = self._core.request_sync("GET", "/agentmemory/livez")
+        return body  # type: ignore[no-any-return]
+
+    def remember(
+        self,
+        *,
+        content: str,
+        project: Optional[str] = None,
+        title: Optional[str] = None,
+        concepts: Optional[List[str]] = None,
+        type: Optional[str] = None,
```

Ruff (0.15.12)

[error] Function argument `type` is shadowing a Python builtin

(A002)